src/Spice86.Core/Emulator/Devices/Input/Keyboard/KeyboardPorts.cs

```diff
@@ -0,0 +1,22 @@
+namespace Spice86.Core.Emulator.Devices.Input.Keyboard; 
+
+/// <summary>
+/// IDs of keyboard ports for both read and write. <br/>
+/// https://www.win.tue.nl/~aeb/linux/kbd/scancodes-11.html
+/// </summary>
+public static class KeyboardPorts {
+    /// <summary>
+    /// Port used by the CPU to read the input buffer, or write to the output buffer, of the keyboard controller.
+    /// </summary>
+    public const byte IOPort = 0x60;
+
+    /// <summary>
+    /// Port that can be used by the CPU to read the status register of the keyboard controller.
+    /// </summary>
+    public const byte StatusRegister = 0x64;
+
+    /// <summary>
+    /// If the CPU writes to port 0x64, the byte is interpreted as a command byte. 
+    /// </summary>
+    public const byte CommandPort = 0x64;
```

Comment:
It's a bit confusing to have 2 names for the same port value. Do we have other devices like this?

Reply:
Not really. Maybe VGA ports ?
This is the same as here:
https://github.com/sarah-walker-pcem/pcem/blob/dev/src/keyboard/keyboard_at.c#L314

Reply:
Yeah, I have VGA ports with different names for reading and writing when they're being used in read-only or write-only context.
To me different names for the same port in different contexts is fine.